<ui version="4.0" >
 <class>UploadPreferences</class>
 <widget class="QWidget" name="UploadPreferences" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>237</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QListView" name="profilesList" />
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="KPushButton" name="addProfileButton" >
       <property name="text" >
        <string>&amp;Add Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="removeProfileButton" >
       <property name="text" >
        <string>&amp;Remove Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="modifyProfileButton" >
       <property name="text" >
        <string>&amp;Modify Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>profilesList</tabstop>
  <tabstop>addProfileButton</tabstop>
  <tabstop>removeProfileButton</tabstop>
  <tabstop>modifyProfileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
